src/scss/grommet-core/_objects.form.scss

```diff
@@ -73,4 +73,8 @@
 
 .#{$grommet-namespace}form--compact {
   max-width: $compact-form-width;
+
+  // Fixes a bug in Windows which causes overflow.
+  .#{$grommet-namespace}check-box__label { width: calc(100% - 36px); }
```

Comment:
I'm worried with the fixed 36px in there.

I propose we convert the label to a flexbox element. Ideas?
![screen shot 2017-09-08 at 7 52 42 am](https://user-images.githubusercontent.com/1207250/30390630-99a442c4-986b-11e7-95ba-495f2ceb724d.png)

Reply:
This makes sense, I'll cross browser test this to verify it holds up.

Reply:
@alansouzati the proposed styles seem to work well across browsers. I had to add a small fix to the radio button which was losing its width but this should be good to go. 🎊